pkg/mongoproxy/plugins/schema/schema.go

```diff
@@ -145,7 +155,7 @@ func (p *SchemaPlugin) Configure(d bson.D) error {
 	if err := watcher.Add(path.Dir(p.conf.SchemaPath)); err != nil {
 		return err
 	}
-
+	<-done
```

Comment:
have we tested it with sending queries to local mongodb? if my reading/understanding is correct, this `done` will block `SchemaPlugin.Configure()` return, thus no query could be processed?